beacon/beacon.go

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (C) 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+// Package beacon contains an implementation of GA4GH Beacon API (http://ga4gh.org/#/beacon).
+package beacon
+
+import (
+	"encoding/xml"
+	"fmt"
+	"html/template"
+	"net/http"
+	"strconv"
+
+	"github.com/googlegenomics/beacon-go/internal/variants"
+	"google.golang.org/appengine"
+)
+
+const beaconAPIVersion = "v0.0.1"
+
+var (
+	aboutTemplate = template.Must(template.ParseFiles("about.xml"))
+)
+
+// Server implements a GA4GH Beacon API (https://github.com/ga4gh-beacon/specification/blob/master/beacon.md) backed
```

Comment:
Can you move this link into the package comment and elide it here?

// Package beacon implements the GA4GH Beacon API (https://github.com/ga4gh-beacon/specification/blob/master/beacon.md).
package beacon

// Server provides handlers for Beacon API requests.

Reply:
done